### ISSUE TITLE
As a user, I want to hide assets I can't work on

### ISSUE THREAD

**Issue by lalgee** [LibraryOfCongress/concordia#702]
**Is your feature request related to a problem? Please describe.**
From History Hub user Beth - https://historyhub.history.gov/thread/3114 - Is there a way to filter out work I've already done when I'm searching for new stuff to review? (I'm wasting time having to click and reclick to get to work to review, and in some cases, am finding stuff I've already worked on.)

User story:  As a user, when looking for a page to work on, I want to be able to filter to show only the pages I have worked on or only those I have not.  When I review an item, and make edits I then can't tell that "In Review" item apart from others I haven't worked on.  Therefore I click on items to review and can't because I was the most recent person to  edit them.

**Describe the solution you'd like**
Implement filtering on the project, item, and asset pages for registered users to allow them to only show items they are able to work on (by filtering out in-review items that they were the last volunteer to edit).  Since this is yes/no it could be a toggle, rather than a filter menu.

**Describe alternatives you've considered**

**Additional context**
Different but related feature we might want to implement at the same time - filter to show all the items a registered users has ever worked on.  This would serve a similar function to the links in "My Contributions" on their profile but might be more user friendly, especially for folks who have worked on many items/assets.

**Comment by acdha** [comment]
There's a performance issue with this we'll want to be careful about — currently those pages are cached so they load faster but adding per-user filtering means we have to be careful to keep those pages private to the user looking at them.

This also pulls some of the broader UI discussion we've been having about how to allow people to favor a particular mode of activity since this seems like something we'd want to also have reflected on the asset page itself (e.g. letting people navigate easily from reviewable page to reviewable page rather than the next transcribable page) and the find-next-page button, especially if #671 adds that in more places.

**Comment by VVH** [comment]
yes, I agree that these are linked issues and something we'll want to address carefully as I see it could be complicated. It's key to positive user experience so hopefully we can move on this in the new year.

**Comment by lalgee** [comment]
Related to feature request identified by Winter 2020 user research: "Remember which items a user has worked on, and what they did (Transcribe, edit, review, tag…)"

**Comment by lalgee** [comment]
Related to https://github.com/LibraryOfCongress/concordia/issues/808 and not offering users activities they can't do.

**Comment by jkueloc** [comment]
Moved to Jira: [CONCD-64](https://staff.loc.gov/tasks/browse/CONCD-64)
